src/server/game/Entities/Player/KillRewarder.cpp

```diff
@@ -232,17 +232,9 @@ void KillRewarder::_RewardGroup()
 
             // 3.1.3. Reward each group member (even dead or corpse) within reward distance.
             for (GroupReference* itr = _group->GetFirstMember(); itr != nullptr; itr = itr->next())
-            {
```

Comment:
please revert all the codestyle changes, they make it harder to review what was actually changed

Reply:
But this brackets become unnecessary with line removal

Reply:
Maybe keep at least the `for` loop brackets, since there is more than 1 line inside that loop.